js/core/component.js

```diff
@@ -504,6 +504,12 @@ var Component = Class.inherit({
         return deprecatedOptions.hasOwnProperty(name);
     },
 
+    _setOptionSilent: function(name, value) {
+        this._cancelOptionChange = name;
```

Comment:
Why is option name used instead of boolean flag?

Reply:
This made for this case:

```
_setOptionSilent: function(name, value) {
   this._cancelOptionChange = true;
   this.option(name, value);  // when more than one option will be changed on this step (for example, as the result of the binding), they should not be changed silently
   this._cancelOptionChange = false;
}
```